src/NzbDrone.Core/Notifications/NotificationBase.cs

```diff
@@ -72,14 +88,14 @@ public override string ToString()
 
         private bool HasConcreteImplementation(string methodName)
         {
-            var method = GetType().GetMethod(methodName);
+            var methods = GetType().GetMethods().Where(method => method.Name == methodName).FirstOrDefault();
```

Comment:
I think `method` is correct for the variable name, since you only ever get one.

Reply:
this isn't strictly meant to be .FirstOrDefault() as onDelete should always be 2 - it's one of the things I need to sort with it

Reply:
This has been altered - @markus101 can I get some feedback on this one please as it's a larger change